### PR TITLE
Add new child resources IPAM + VNV to existing CLI ext 'virtual-network-manager'

### PR DESCRIPTION
**Pull request by danlope**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py [hunk 1]
```diff
@@ -157,6 +158,7 @@ def test_network_manager_security_admin_config_crud(self, resource_group):
                  '--scope-accesses "SecurityAdmin" "Connectivity" '
                  '--network-manager-scopes subscriptions={sub} -l eastus2 --resource-group {rg}')
 
+        #todo
```
Comment (danlope) [inline src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py:161]: todo?

File: src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py [hunk 2]
```diff
@@ -213,19 +215,19 @@ def test_network_manager_admin_rule_crud(self, virtual_network, resource_group):
                  '--applies-to-groups network-group-id={sub}/resourceGroups/{rg}/providers/Microsoft.Network/networkManagers/{manager_name}/networkGroups/{group_name}')
 
         self.cmd('network manager security-admin-config rule-collection rule create -g {rg} --network-manager-name {manager_name} --configuration-name {config_name} --rule-collection-name {collection_name} '
-                 '--rule-name {rule_name} --kind "Custom" --protocol "Tcp" --access "Allow" --priority 32 --direction "Inbound"',
+                 '--rule-name {rule_name} --protocol "Tcp" --access "Allow" --priority 32 --direction "Inbound"',
```
Comment (danlope) [inline src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py:218]: todo: kind?

**Comment by AllyW** [comment]
> ❌Azure CLI Extensions Breaking Change Test
> > ❌network-manager
> > 
> > 
> > 
> > rule
> > cmd_name
> > rule_message
> > suggest_message
> > 
> > 
> > 
> > 
> > ❌ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > network manager security-admin-config create
> > cmd `network manager security-admin-config create` update parameter `network_manager_name`: updated property `options` from `['--name', '--network-manager-name', '-n']` to `['--manager-name', '--network-manager-name']`
> > please change property `options` from `['--manager-name', '--network-manager-name']` to `['--name', '--network-manager-name', '-n']` for parameter `network_manager_name` of cmd `network manager security-admin-config create`
> > 
> > 
> > ❌ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > network manager security-admin-config update
> > cmd `network manager security-admin-config update` update parameter `network_manager_name`: updated property `options` from `['--name', '--network-manager-name', '-n']` to `['--manager-name', '--network-manager-name']`
> > please change property `options` from `['--manager-name', '--network-manager-name']` to `['--name', '--network-manager-name', '-n']` for parameter `network_manager_name` of cmd `network manager security-admin-config update`
> > 
> > 
> > ⚠️ [1011 - SubgroupAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1011.md)
> > network manager ipam-pool
> > sub group `network manager ipam-pool` added
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > network manager security-admin-config create
> > cmd `network manager security-admin-config create` added parameter `network_group_address_space_aggregation_option`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > network manager security-admin-config create
> > cmd `network manager security-admin-config create` update parameter `configuration_name`: updated property `options` from `['--configuration-name']` to `['--configuration-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > network manager security-admin-config update
> > cmd `network manager security-admin-config update` added parameter `network_group_address_space_aggregation_option`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > network manager security-admin-config update
> > cmd `network manager security-admin-config update` update parameter `configuration_name`: updated property `options` from `['--configuration-name']` to `['--configuration-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1011 - SubgroupAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1011.md)
> > network manager verifier-workspace
> > sub group `network manager verifier-workspace` added

@danlope  your pr removed option `-n` for parameter `network_manager_name` of cmd `network manager security-admin-config create`. This is a breaking change and please confirm this change.

File: src/network-manager/azext_network_manager/aaz/latest/network/manager/ipam_pool/_create.py [hunk 3]
```diff
@@ -72,10 +72,9 @@ def _build_arguments_schema(cls, *args, **kwargs):
             arg_group="Body",
             help="The geo-location where the resource lives",
             required=True,
-            # todo: confirm
-            # fmt=AAZResourceLocationArgFormat(
-            #     resource_group_arg="resource_group",
-            # ),
+            fmt=AAZResourceLocationArgFormat(
```
Comment (danlope) [inline src/network-manager/azext_network_manager/aaz/latest/network/manager/ipam_pool/_create.py:75]: Remove

**Comment by necusjz** [comment]
> Hi @danlope
> 
> ## Release Suggestions
> ### Module: network-manager
> * Update `VERSION` to `1.3.0` in `src/network-manager/setup.py`
> 
> #### Notes
> * For more info about extension versioning, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

we need to update `setup.py` to release an extension with new version

File: src/network-manager/azext_network_manager/aaz/latest/network/manager/ipam_pool/_create.py (hunk 3 above)
Comment (danlope) [reply src/network-manager/azext_network_manager/aaz/latest/network/manager/ipam_pool/_create.py:75]: Nvm - spoke with Arjun, it is okay to use resource group region if not provided by user. As long as it tells user this behavior using --help command, which it does:

image

**Review by necusjz — APPROVED** [review]

**Pull request merged**